cmd/server.go

```diff
@@ -135,6 +137,7 @@ const (
 	DefaultADBasicPassword              = ""
 	DefaultADHostname                   = "dev.azure.com"
 	DefaultAutoplanFileList             = "**/*.tf,**/*.tfvars,**/*.tfvars.json,**/terragrunt.hcl,**/.terraform.lock.hcl"
+	DefaultAllowCommands                = "plan,apply,unlock"
```

Comment:
What about version?

Are there possibly other commands too that we're missing?

What was the reason for not including import here?

Reply:
`version` is available by default on NewCommentParser. If this is confusing, it might be better to add them.

I guess this default value provides our general atlantis use-case. And I think these values are enough for users who starts atlantis first time.

Reply:
based on https://github.com/runatlantis/atlantis/pull/2877#discussion_r1056875542, I will add `version` and `approve_policies` into here 👌

Reply:
Lets be explicit with version so its consistent, no?

Reply:
true!